episodes/17-live.md

```diff
@@ -90,9 +106,9 @@ Some challenges are:
 
 ::::::::::::::::::::::::::::::::::::::::::::::::::
 
-Live coding fits well into the practice-feedback model we have been discussing - by providing
-learners with continuous opportunities for practice (every time they type in a line of code)
-and continuous feedback (their code either works or fails with an error message). It is
+Live demos fits well into the practice-feedback model we have been discussing - by providing
+learners with continuous opportunities for practice (every time they type a line of code or perform a procedure)
+and continuous feedback (their attempt either works or fails with an error message). It is
```

Comment:
the "error message" part may not translate to non-coding contexts, but i'm not sure what the best fix is here.

Reply:
Error messages defnitely come up in non-coding contexts IME.